packages/shared-ux/avatar/user_profile/impl/user_profile.ts

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { VISUALIZATION_COLORS } from '@elastic/eui';
+
+/**
+ * IMPORTANT:
+ *
+ * The types in this file have been imported from
+ * `x-pack/plugins/security/common/model/user_profile.ts`
+ *
+ * When making changes please ensure to keep both files in sync.
+ */
+
+/**
+ * Describes basic properties stored in user profile.
+ */
+export interface UserProfile<D extends UserProfileData = UserProfileData> {
+  /**
+   * Unique ID for of the user profile.
+   */
+  uid: string;
+
+  /**
+   * Indicates whether user profile is enabled or not.
+   */
+  enabled: boolean;
+
+  /**
+   * Information about the user that owns profile.
+   */
+  user: UserProfileUserInfo;
+
+  /**
+   * User specific data associated with the profile.
+   */
+  data: Partial<D>;
+}
+
+/**
+ * Basic user information returned in user profile.
+ */
+export interface UserProfileUserInfo {
+  /**
+   * Username of the user.
+   */
+  username: string;
+  /**
+   * Optional email of the user.
+   */
+  email?: string;
+  /**
+   * Optional full name of the user.
+   */
+  full_name?: string;
```

Comment:
This is weird, we use camel-case for variable names. Is this due to  naming in `x-pack/plugins/security/common/model/user_profile.ts`?

Reply:
[b448ed9](https://github.com/elastic/kibana/pull/139247/commits/b448ed964fc7ea20546b4e23180bff4253eaa96b) for changes thanks!